### PR TITLE
Feature 3

### PR DESCRIPTION
**Pull request by guarani-frba-utn-edu-ar**
Added func anda paragraph for feature-3

### CODE REVIEW

**Review by Matestampa — CHANGES_REQUESTED** [review]
Ponele un comment

File: app.js [hunk 1]
```diff
@@ -14,3 +14,8 @@ function feature_2(){
     console.log("commit 1 del colab papa"); //el comentario
 
 }
+
+function feature_3(){
+   console.log("commit 1 del colab ");
```
Comment (Matestampa) [inline app.js]: Aca seria el comment

**Review by Matestampa — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (guarani-frba-utn-edu-ar) [reply app.js]: Ok ya lo hago

**Review by Matestampa — APPROVED** [review]
Perfecto!!!

**Pull request merged**